nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/pom.xml

```diff
@@ -355,10 +355,5 @@
             <artifactId>spock-core</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
hmm I didn't directly remove this perhaps odd merge during rebase? Spock tests are still functioning however I will restore it just in case.